pkg/linode-bs/deploy/kubernetes/06-ss-csi-linode-controller.yaml

```diff
@@ -44,7 +59,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: linode-csi-plugin
-          image: linode/linode-blockstorage-csi-driver:{{ .Values.image.tag }}
```

Comment:
These manifests are a template for this script: https://github.com/linode/linode-blockstorage-csi-driver/blob/master/hack/release-yaml.sh

So, this value should be left as-is

Reply:
Reverted the change 👍